test/packaging/ansible/shutdown-and-terminate.yml

```diff
@@ -53,109 +51,78 @@
       vars:
         display_name: "{{ iid }}:{{ inventory_hostname }}"
 
-    - name: Assert service
+    - name: assert service
       include_role:
         name: assert-service
 
-- name: controlled shutdown behaviour
+- name: test agent behaviour on host shutdown
   hosts: testing_hosts
   vars:
-    minutes_until_alert_opens: 10
-    minutes_until_alert_closes: 10
-    minutes_to_send_data: 3
-    # Add here name of the instances instances that doesn't support Smart HNR (shutdown detection)
+    # Add here hosts of the instances that doesn't support Smart HNR (shutdown detection) e.g. - "amd64:ubuntu14.04"
     instances_not_supporting_shutdown_detection: {
-      # e.g. - "amd64:ubuntu14.04"
+      "amd64:debian-jessie"
     }
-    env_vars:
+
+    host_supports_shutdown_detection: >-
+      {{
+        true if inventory_hostname not in instances_not_supporting_shutdown_detection
+        else false
+      }}
+
   tasks:
-    - name: Delete HNR alerts
-      include_role:
-        name: infrastructure-alert-delete
-    - name: Running tests
-      block:
-        - name: Create HNR alerts
-          include_role:
-            name: infrastructure-alert-create
-
-        - name: Pause for few minutes to let the agent send some data
-          pause:
-            minutes: "{{ minutes_to_send_data }}"
-
```

Comment:
Int he previous test it was also tested that the alert is triggered when the agent is stopped. Is there any reason why this is not tested now?